library/spdm_responder_lib/libspdm_rsp_algorithms.c

```diff
@@ -631,12 +804,46 @@ libspdm_return_t libspdm_get_response_algorithms(libspdm_context_t *spdm_context
         }
     }
 
+    /* if both PQC and tradition are enabled, choose PQC and disable tradition */
```

Comment:
>If both PQC and traditional are enabled, choose PQC and disable traditional.

Reply:
This comment is just `tradition` -> `traditional`.

Reply:
updated.